omnibus/config/software/python3.rb

```diff
@@ -65,12 +65,18 @@
   end
 
 else
-  dependency "vc_redist_14"
-  default_version "3.7.1"
+  if windows_arch_i386?
+    default_version "3.7.3"
```

Comment:
why not ship the same version between 32 and 64 bits ?

Reply:
I needed a newer version to get binary wheels for some of the python deps.

Reply:
should we update the 64bits to have the same version ?

Reply:
I will.  We'll need to generate a new python zip for the next release, anyway, for a different card, so we'll catch up then.  Don't want to make that change at the last minute